hatchery/hatchery.go

```diff
@@ -425,15 +425,15 @@ func launch(w http.ResponseWriter, r *http.Request) {
 		}
 		newItem, err := createGen3LicenseUserMap(dbconfig, userName, nextLicenseId, Config.ContainersMap[hash])
 		if err != nil {
-			Config.Logger.Printf(err.Error())
+			Config.Logger.Print(err.Error())
 		}
 		Config.Logger.Printf("Created new license-user-map item: %v", newItem)
 
 	}
 
 	allpaymodels, err := getPayModelsForUser(userName)
 	if err != nil {
-		Config.Logger.Printf(err.Error())
+		Config.Logger.Print(err.Error())
```

Comment:
If linting issues are also being addressed, shouldn't Line:495 also use `Print` instead of `Printf`?

Reply:
That one has been fixed too